core/src/main/java/com/arcrobotics/ftclib/gamepad/TriggerReader.java

```diff
@@ -26,21 +31,35 @@ public TriggerReader(GamepadEx gamepad, GamepadKeys.Trigger trigger) {
         this.gamepad = gamepad;
         this.trigger = trigger;
 
-        if (this.gamepad.getTrigger(trigger) > 0.5) {
+        if (this.gamepad.getTrigger(trigger) > this.threshold) {
             currState = true;
         } else {
             currState = false;
         }
 
```

Comment:
This is mostly redundant, since it's the exact same as the other constructor, it just doesn't set a value. I would replace this with just `this(gamepad, trigger, 0.5);`